Importer/jctdata/jac.py

```diff
@@ -133,9 +133,40 @@ def issn_clusters(coincident_issn_files, clusters_file):
     issn_clusters.sort()
     issn_clusters = list(k for k,_ in itertools.groupby(issn_clusters))
 
+    d = {}
+    for row in issn_clusters:
+        if row[0] not in d:
+            d[row[0]] = []
+        d[row[0]] += [row[x + 1] for x in range(len(row) - 1)]
```

Comment:
```suggestion
        d[row[0]] += row[1:]
```
Also, will this list need to be deduplicated?

Reply:
The list doesn't need to be deduplicated at this stage - they are already deduplicated in the previous stage of the process, and they'll get deduplicated again later in this stage too